plugins/search-backend-module-elasticsearch/src/alpha.ts

```diff
@@ -68,6 +68,13 @@ export default createBackendModule({
         config: coreServices.rootConfig,
       },
       async init({ searchEngineRegistry, logger, config }) {
+        const baseKey = 'search.elasticsearch';
+        const baseConfig = config.getOptional(baseKey);
+        if (!baseConfig) {
+          logger.info('No configuration found under "search.elasticsearch" key.  Skipping search engine inititalization.');
```

Comment:
```suggestion
          logger.warn('No configuration found under "search.elasticsearch" key.  Skipping search engine inititalization.');
```

Feel like warning is better, will jump out at them they missed something.

Reply:
Done!  Fixed a prettier check as well